src/os_esp32.cc

```diff
@@ -50,7 +53,7 @@ namespace toit {
 
 // Flags used to get memory for the Toit heap, which needs to be fast and 8-bit
 // capable.
-static const int TOIT_HEAP_CAPS_FLAGS = MALLOC_CAP_8BIT | MALLOC_CAP_INTERNAL | MALLOC_CAP_DMA;
+static int toit_heap_caps_flags = 0;
```

Comment:
Extend comment to tell that this is dynamically computed and updated based on the capabilities of the platform we run on.